cli/cmd/helm/version.go

```diff
@@ -0,0 +1,27 @@
+package helm
+
+import (
+	"fmt"
+	"github.com/spf13/cobra"
+	"github.com/timescale/tobs/cli/cmd"
+)
+
+// uninstallCmd represents the uninstall command
+var versionCmd = &cobra.Command{
+	Use:   "version",
+	Short: "Version of tobs",
+	Args:  cobra.ExactArgs(0),
+	RunE:  version,
+}
+
+func init() {
+	cmd.RootCmd.AddCommand(versionCmd)
+}
+
+const tobsVersion = "0.4.0"
```

Comment:
There is no way currently to understand what version of `tobs` a user is using. So adding a new `version` cmd to know the tobs version being used.

Reply:
Automatically in the sense? From the CI during releases?